image_width: 848
image_height: 800
camera_matrix: #K
  rows: 3
  cols: 3
  data: [1460.5480364135758, 0.0, 348.6581998411612, 0.0, 1369.21519471933, 376.2039040798353, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-5.074147363843537, 45.45912964059443, -0.008268488080948886, -0.0344104458271585, -193.67356950083155]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [1460.5480364135758, 0.0, 348.6581998411612, 0.0, 0.0, 1369.21519471933, 376.2039040798353, 0.0, 0.0, 0.0, 1.0, 0.0]
